test/mechanics/integrands/HistoryData.cpp

```diff
@@ -42,9 +43,9 @@ class HistoryDataContiguousMemory
     //! @param cellNum: Number of the cell containing the integration point
     //! @param ipNum: Number of the integration point
     //! @return History data of the integration point
-    const T& GetIPHistoryData(const unsigned int cellNum, const unsigned int ipNum) const
+    const T& GetIpHistoryData(const unsigned int cellNum, const unsigned int ipNum) const
     {
-        assert(cellNum < mHistoryData.size() / mIpsPerCell);
+        assert(cellNum < mHistoryData.size() / mIpsPerCell && "Have you initialized the history data?");
```

Comment:
Shouldn't these be exceptions?

Reply:
I thought about it, wasn't sure. What does anybody else think about it?

Reply:
Can we delay everything regarding these containers or the integrands and move the discussion to an `IntegrandLib` issue? (--> #146)